test/testtoken.cpp

```diff
@@ -355,6 +355,22 @@ class TestToken : public TestFixture {
         ASSERT_EQUALS(false, tok.isUtf32());
         ASSERT_EQUALS(false, tok.isLong());
         ASSERT_EQUALS(true, tok.isCMultiChar());
+
+        tok.str("'\''");
+        ASSERT_EQUALS(true, tok.isCChar());
+        ASSERT_EQUALS(false, tok.isUtf8());
+        ASSERT_EQUALS(false, tok.isUtf16());
+        ASSERT_EQUALS(false, tok.isUtf32());
+        ASSERT_EQUALS(false, tok.isLong());
+        ASSERT_EQUALS(false, tok.isCMultiChar());
+
+        tok.str("'\r\n'");
```

Comment:
I have the feeling this string should be 

    "'\\r\\n'"

the string length should be 6